client/themes.ts

```diff
@@ -1,4 +1,21 @@
-import { createTheme } from '@mui/material/styles'
+import { createTheme, PaletteColorOptions } from '@mui/material/styles'
+
+declare module '@mui/material/styles' {
+  interface CustomPalette {
+    OKFNCoolGray: PaletteColorOptions
+    OKFNBlue: PaletteColorOptions
+  }
+  interface Palette extends CustomPalette {}
+  interface PaletteOptions extends CustomPalette {}
+}
+
+declare module '@mui/material/Button' {
+  // <-- Added `/Button` here
+  interface ButtonPropsColorOverrides {
+    OKFNCoolGray: true
```

Comment:
With this we can now use any colors from the designs by just adding them here and then in a component with the `color` prop.